src/modules/localStorage.js

```diff
@@ -0,0 +1,129 @@
+class Storage {
+  static getToDo() {
+    let todoL;
+    if (!localStorage.getItem('todoL')) {
+      todoL = [];
+    } else {
+      todoL = JSON.parse(localStorage.getItem('todoL'));
+    }
+    return todoL;
+  }
+
+  static addTodo(todo) {
+    const todoL = Storage.getToDo();
+
+    todoL.push(todo);
+
+    localStorage.setItem('todoL', JSON.stringify(todoL));
+  }
+
+  static remove(id) {
+    const todoL = Storage.getToDo();
+    id = Number(id);
+    todoL.forEach((todo, i) => {
+      if (todo.id === id) {
+        todoL.splice(i, 1);
+      }
+    });
+    localStorage.setItem('todoL', JSON.stringify(todoL));
+    Storage.resetId();
+  }
+
+  static resetId() {
+    const todoL = Storage.getToDo();
+    const arr = [];
+
+    todoL.forEach((item) => {
+      const newId = { ...item, id: arr.length + 1 };
+      arr.push(newId);
+      localStorage.setItem('todoL', JSON.stringify(arr));
+    });
+  }
+
+  static checkboxCompleted(id, status) {
+    const todoL = Storage.getToDo();
+    id = Number(id.textContent);
+
+    todoL.forEach((x) => {
+      if (x.id === id) {
+        if (status) {
+          x.completed = true;
+        } else {
+          x.completed = false;
+        }
+      }
+      localStorage.setItem('todoL', JSON.stringify(todoL));
+    });
+    return todoL;
+  }
+
+  static removeCompleted() {
+    const todoL = Storage.getToDo();
+
+    const notCompleted = todoL.filter((x) => x.completed === false);
+    localStorage.setItem('todoL', JSON.stringify(notCompleted));
+    Storage.resetId();
+    window.location.reload();
+  }
+
+  static deleteCompletedTask() {
+    const todoL = Storage.getToDo();
+
+    const notCompleted = todoL.filter((x) => x.completed === false);
+    localStorage.setItem('todoL', JSON.stringify(notCompleted));
+    Storage.resetId();
+    // window.location.reload();
```

Comment:
- [ ] Kindly remove all commented-out code that is un used/ unwanted from your code base. It makes it look cluttered and messy.

Reply:
Hello @Nemwel-Boniface, 
Sure going to make the suggested changes.
Thank you.